packages/tizen_app_manager/tizen/src/tizen_app_manager_plugin.cc

```diff
@@ -98,24 +49,68 @@ void TizenAppManagerPlugin::HandleMethodCall(
 void TizenAppManagerPlugin::RegisterObserver(
     std::unique_ptr<flutter::EventSink<flutter::EncodableValue>> &&events) {
   LOG_DEBUG("RegisterObserver");
-  int ret = app_manager_set_app_context_event_cb(ContextEventCB, (void *)this);
+  int ret = app_manager_set_app_context_event_cb(
+      [](app_context_h app_context, app_context_event_e event,
+         void *user_data) {
+        if (app_context == nullptr) {
+          return;
+        }
+        char *app_id = nullptr;
+        app_context_h clone_context = nullptr;
+        int ret = app_context_get_app_id(app_context, &app_id);
+        if (ret != APP_MANAGER_ERROR_NONE || app_id == nullptr) {
+          LOG_ERROR("get app Id error! : %s", get_error_message(ret));
+          if (app_id) {
+            free(app_id);
+          }
+          return;
+        }
+
+        ret = app_context_clone(&clone_context, app_context);
+        if (ret != APP_MANAGER_ERROR_NONE || clone_context == nullptr) {
+          LOG_ERROR("clone context error! : %s", get_error_message(ret));
+          free(app_id);
+          return;
+        }
+        int int_address = std::intptr_t(clone_context);
+        LOG_INFO("event: %d, app_id: %s", event, app_id);
+        LOG_INFO("app_context: %p, clone_context: %p, int_addr: %p",
+                 app_context, clone_context, int_address);
+        TizenAppManagerPlugin *plugin =
+            static_cast<TizenAppManagerPlugin *>(user_data);
+        flutter::EncodableMap msg;
+        msg.insert(std::pair<flutter::EncodableValue, flutter::EncodableValue>(
+            "appId", std::string(app_id)));
+        msg.insert(std::pair<flutter::EncodableValue, flutter::EncodableValue>(
+            "handle", int_address));
+        if (event == APP_CONTEXT_EVENT_LAUNCHED &&
+            plugin->launch_events_ != nullptr) {
```

Comment:
Prefer if (value) over if (value != nullptr).

Reply:
modified to if(value) form. Thank you!